app/Services/Scans/ScanFlowService.php

```diff
@@ -251,20 +251,22 @@ public function resolveNextUrl(): string
                 DiscordNotifier::init()->notify("Next sub step doesn't belong to next step! Step ID: {$nextStep->id}. Sub step ID: {$nextSubStep->id}.");
                 $nextUrl = '';
             } else {
-                $nextUrl = route('cooperation.frontend.tool.quick-scan.index',
-                    ['cooperation' => $cooperation, 'step' => $nextStep, 'subStep' => $nextSubStep]);
+                $nextUrl = route('cooperation.frontend.tool.simple-scan.index', [
+                    'cooperation' => $cooperation, 'scan' => $this->scan, 'step' => $nextStep, 'subStep' => $nextSubStep
+                ]);
             }
         } elseif ($nextStep instanceof Step && $nextQuestionnaire instanceof Questionnaire) {
             if ($nextQuestionnaire->step_id !== $nextStep->id) {
                 // TODO: Temporary, remove if when no issues arise
                 DiscordNotifier::init()->notify("Next questionnaire doesn't belong to next step! Step ID: {$nextStep->id}. Questionnaire ID: {$nextQuestionnaire->id}.");
                 $nextUrl = '';
             } else {
-                $nextUrl = route('cooperation.frontend.tool.quick-scan.questionnaires.index',
-                    ['cooperation' => $cooperation, 'step' => $nextStep, 'questionnaire' => $nextQuestionnaire]);
+                $nextUrl = route('cooperation.frontend.tool.questionnaires.index', [
+                    'cooperation' => $cooperation, 'scan' => $this->scan, 'step' => $nextStep, 'questionnaire' => $nextQuestionnaire
+                ]);
             }
         } else {
-            $nextUrl = route('cooperation.frontend.tool.quick-scan.my-plan.index', ['cooperation' => $cooperation]);
+            $nextUrl = route('cooperation.frontend.tool.my-plan.index', ['cooperation' => $cooperation]);
```

Comment:
Not consistent, see https://github.com/Ecodenl/Woningdossier/pull/1254/files#diff-779e740370473e63af8a00ccfcac693843e18b3cf782c3324470079d03d9979aR60
Different route